src/cmd-buildextend-live

```diff
@@ -414,8 +414,13 @@ def generate_iso():
         os.makedirs(os.path.join(tmpisoroot, 'boot/grub'))
         # can be EFI/fedora or EFI/redhat
         grubpath = glob.glob(os.path.join(tmpisoroot, 'EFI/*/grub.cfg'))
-        shutil.move(grubpath[0], os.path.join(tmpisoroot, 'boot/grub/grub.cfg'))
-
+        boot_grubpath = os.path.join(tmpisoroot, 'boot/grub/grub.cfg')
+        shutil.move(grubpath[0], boot_grubpath)
+        for key in list(files_with_karg_embed_areas):
+            if key == grubpath:
```

Comment:
Sorry, I made a mistake in my earlier suggestion. This should indeed be `grubpath[0]` as @mike-nguyen pointed out to me.

Alternatively, we can also do

```python
grubpath = grubpath[0]
```

higher up after the `glob.glob()` line. There should only be a single result, and we could additionally assert that the list length is 1, but will leave that up to you. :)

Reply:
135cac97c434c7e5bca061487350f42d9af587b4